### DISCUSSION TITLE
The used service does not resize the image.

### DISCUSSION THREAD

**Discussion by basti030** [FriendsOfShopware/FroshPlatformThumbnailProcessor#130] [Patterns]
The used service does not resize the image. sample url: https://domain.de/media/70/70/b3/1712654100/froshthumbnailprocessortestimage.jpg?width=200

What are we doing wrong?

**Comment by tinect** [comment]
I don‘t know. you might want to ask the service provider.

**Reply by basti030** [reply]
Unfortunately, we have a managed server and have to take care of everything ourselves! The CDN provider won't help, right?

**Comment by tinect** [comment]
yes, you need to use a service which can scale the images. 
if you use selfhosted, like imgproxy, you need to cache the results on your server otherwise you will your server by regenerating all the time.

but if you have a cdn provider, you can check if it handle the scaling.
